src/main/kotlin/us/timinc/mc/cobblemon/spawnnotification/SpawnNotification.kt

```diff
@@ -110,6 +121,13 @@ object SpawnNotification : ModInitializer {
             ))
 
             Broadcast.broadcastMessage(messageComponent)
+        }
+
+        // if config.limitedRange && player.inRange()?
+        if (config.limitRange) {
```

Comment:
There's some logic reconciliation to do here. If the config has a limited range, across dimensions should probably be treated as false. Biome/coords are fine though. Thoughts?

Reply:
Actually now that you mention it, if announceCrossDimensions = true, I'm now causing two broadcastMessages to trigger at once. Let me fix that as well as addressing your remark